go/test/endtoend/cluster/mysqlctl_process.go

```diff
@@ -236,11 +237,17 @@ func (mysqlctl *MysqlctlProcess) Connect(ctx context.Context, username string) (
 // MysqlCtlProcessInstanceOptionalInit returns a Mysqlctl handle for mysqlctl process
 // configured with the given Config.
 func MysqlCtlProcessInstanceOptionalInit(tabletUID int, mySQLPort int, tmpDirectory string, initMySQL bool) *MysqlctlProcess {
+	var initFile = path.Join(os.Getenv("VTROOT"), "/config/init_db.sql") //default value
+	if isSQL, err := isSQLFlavor(); err == nil {
+		if !isSQL {
+			initFile = path.Join(os.Getenv("VTROOT"), "config/init_maria_db.sql")
+		}
+	}
```

Comment:
This is not about SQL. I think we should also get rid of the MariaDB code. We also have a flavor package to use for this.